src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -125,10 +125,10 @@ void fillInnerParts() {
         applicantListPanel = new ApplicantListPanel(logic.getFilteredPersonList());
         personListPanelPlaceholder.getChildren().add(applicantListPanel.getRoot());
 
-        positionListPanel = new PositionListPanel(null); // TO LINK WITH LOGIC
+        positionListPanel = new PositionListPanel(logic.getFilteredPositionList()); // TO LINK WITH LOGIC
```

Comment:
Would removing the comment be better? I presume it's already linked with logic in this PR